dex/networks/eth/txdata.go

```diff
@@ -128,31 +234,44 @@ func ParseRefundData(calldata []byte) ([32]byte, error) {
 	return secretHash, nil
 }
 
-// PackInitiateData converts a list of swapv0.ETHSwapInitiation to call data for the
-// initiate function.
-func PackInitiateData(initiations []swapv0.ETHSwapInitiation) ([]byte, error) {
+// packInitiateData converts a list of Initiate to call data for the initiate function.
+func (t *txDataHandlerV0) packInitiateData(initiations []Initiation) ([]byte, error) {
 	parsedAbi, err := abi.JSON(strings.NewReader(swapv0.ETHSwapABI))
```

Comment:
Thoughts about doing this `parsedAbi, err := abi.JSON(strings.NewReader(swapv0.ETHSwapABI))` just once in either an `init()` function or a `txDataHandlerV0` constructor?
It always spits out the same `parsedAbi`, and it's in three different methods.

Reply:
Yes that makes sense.